com.microsoft.mrtk.spatialmanipulation/ObjectManipulator/ObjectManipulator.cs

```diff
@@ -570,106 +639,158 @@ public override void ProcessInteractable(XRInteractionUpdateOrder.UpdatePhase up
+                {
+                    ApplyForcesToRigidbody();
                 }
             }
         }
 
         /// <summary>
-        /// Once the <paramref name="targetPose"/> has been determined, this method is called
+        /// Once the <paramref name="targetTransform"/> has been determined, this method is called
         /// to apply the target pose to the object. Calls <see cref="ModifyTargetPose"/> before
         /// applying, to adjust the pose with smoothing, constraints, etc.
         /// </summary>
-        /// <param name="targetPose">
-        /// The target position, rotation, and scale to set the object to.
-        /// <param/>
-        private void ApplyTargetPose(MixedRealityTransform targetPose)
+        private void ApplyTargetTransform()
         {
             TransformFlags modifiedTransformFlags = TransformFlags.None;
 
-            ModifyTargetPose(ref targetPose, ref modifiedTransformFlags);
+            ModifyTargetPose(ref targetTransform, ref modifiedTransformFlags);
 
             if (rigidBody == null)
             {
                 if (!modifiedTransformFlags.IsMaskSet(TransformFlags.Move))
                 {
-                    HostTransform.position = targetPose.Position;
+                    HostTransform.position = targetTransform.Position;
                 }
                 if (!modifiedTransformFlags.IsMaskSet(TransformFlags.Rotate))
                 {
-                    HostTransform.rotation = targetPose.Rotation;
+                    HostTransform.rotation = targetTransform.Rotation;
                 }
                 if (!modifiedTransformFlags.IsMaskSet(TransformFlags.Scale))
                 {
-                    HostTransform.localScale = targetPose.Scale;
+                    HostTransform.localScale = targetTransform.Scale;
                 }
             }
             else
             {
                 // There is a Rigidbody. Potential different paths for near vs far manipulation
-                if (IsGrabSelected && !useForcesForNearManipulation)
+                if (!useForces)
                 {
-                    rigidBody.MovePosition(targetPose.Position);
-                    rigidBody.MoveRotation(targetPose.Rotation);
+                    rigidBody.MovePosition(targetTransform.Position);
+                    rigidBody.MoveRotation(targetTransform.Rotation);
                 }
-                else
-                {
-                    // We are using forces
-                    rigidBody.velocity = ((1f - Mathf.Pow(moveLerpTime, Time.deltaTime)) / Time.deltaTime) * (targetPose.Position - HostTransform.position);
 
-                    var relativeRotation = targetPose.Rotation * Quaternion.Inverse(HostTransform.rotation);
-                    relativeRotation.ToAngleAxis(out float angle, out Vector3 axis);
+                HostTransform.localScale = targetTransform.Scale;
```

Comment:
Should we also check `modifiedTransformFlags` in this block?

Reply:
According to the comment in `ModifyTargetPose`, the argument `modifiedTransformFlags` is intended only for external modifications like Elastics which are currently not implemented at all. I have no idea how that mechanism would interact with Rigidbody forces, so I'd rather not change that logic and leave it as is for a future overhaul of that mechanism.

From this local code perspective, checking for the flags would at most be a minor optimization.